rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -127,13 +128,16 @@ public Web3Impl(Ethereum eth, RskSystemProperties properties, Wallet wallet) {
             personal_newAccountWithSeed("cow");
```

Comment:
Just so you know, an upcoming PR will move this initialization logic to a brand new `PersonalModule` class that will handle all `personal_*` RPC calls :).

cc @diega.